### ISSUE TITLE
MXNET_NGRAPH_BACKEND=INTERPRETER does not support training

### ISSUE THREAD

**Issue by sfvaroglu** [NervanaSystems/ngraph-mxnet#512]
Hit an error "what():  [13:44:54] src/engine/./threaded_engine.h:380: Unsupported op 'BatchNormTrainingRelu'" when running inception-v3 or inception-v4 with MXNET_NGRAPH_BACKEND=INTERPRETER.

MXNET_NGRAPH_BACKEND=INTERPRETER KMP_AFFINITY=granularity=fine,compact,1,0 OMP_NUM_THREADS=56 python example/image-classification/train_imagenet.py --network inception-v3 --batch-size 128 --num-epochs 1 --num-examples 12850 --num-classes 10 --lr 0.4 --lr-factor 0.1 --lr-step-epochs 30,60,90 --initializer normal --image-shape 3,299,299 --data-train=/dataset/imagenet299px/imagenet_train.rec --data-val=/dataset/imagenet299px/imagenet_val.rec

**Comment by ashokei** [comment]
tested locally with https://github.com/NervanaSystems/ngraph-mxnet/pull/515 , seems ok. please reopen if any issues.
